target/linux/ath79/dts/ar9344_devolo_magic-2-wifi.dts

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,magic-2-wifi", "qca,ar9344";
+
+	aliases {
+		led-boot = &plcr;
+		led-failsafe = &plcr;
+		led-running = &plcw;
+		led-upgrade = &plcr;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan: wlan {
+			label = "devolo:status:wlan";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		plcw: lan_status {
+			label = "devolo:status:dlan";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "netdev";
```

Comment:
You are using 01_leds for this. Don't think the trigger makes sense her.

Reply:
The problem with this, is to find a way to define the eth0 tx to the netdev from dts, still is a netdev. I think we should keep it, there are other devices doing it

Reply:
This default-trigger doesn't work and I think it should either be dropped or actually be made working somehow.
BTW I can't really find existing devices doing the same thing.

Reply:
You are right, I don't see them now, only on the internet...
It is removed now, I didn't find a way to make it work.